samples/xrc/myframe.cpp

```diff
@@ -206,14 +207,17 @@ void MyFrame::OnAnimationCtrlPlay(wxCommandEvent& event)
 
     wxWindow *win = btn->GetParent();
     wxAnimationCtrl *ctrl = XRCCTRL(*win, "controls_animation_ctrl", wxAnimationCtrl);
+    wxGenericAnimationCtrl *generic =
+        XRCCTRL(*win, "controls_generic_animation_ctrl", wxGenericAnimationCtrl);
```

Comment:
I'd probably take this inside `#ifdef wxHAS_NATIVE_ANIMATIONCTRL` as otherwise it's one and the same thing anyhow...